### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by amirkhodabande**

### CODE REVIEW

File: tests/Handlers/Strategies/RequestResponseNamedArgsTest.php [hunk 1]
```diff
@@ -21,6 +21,8 @@ class RequestResponseNamedArgsTest extends TestCase
     private ServerRequestInterface $request;
     private ResponseInterface $response;
 
+    private const LAST_PHP_VERSION_ID = 80000;
```
Comment (akrabat) [inline tests/Handlers/Strategies/RequestResponseNamedArgsTest.php]: Please name as `PHP_8_0_VERSION_ID`

**Review by akrabat — CHANGES_REQUESTED** [review]
In addition to the constant name, please rebase against 4.x and fix conflicts.

**Comment by amirkhodabande** [comment]
> In addition to the constant name, please rebase against 4.x and fix conflicts.

**Review by akrabat — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**